R/getQuote.R

```diff
@@ -373,19 +369,9 @@ getQuote.av <- function(Symbols, api.key, ...) {
     }
     r <- rbind(r, batch.result)
   }
-
-  colnames(r) <- gsub("(^[[:alpha:]])", "\\U\\1", colnames(r), perl = TRUE)
-  r[, "Trade Time"] <- r[, "LastSaleTimestamp"]
-  r[, "LastSaleTimestamp"] <- NULL
-
-  # merge join to produce empty rows for missing results, so that
-  # return value has the same number of rows and order as the input
-  if(length(Symbols) != NROW(r)) {
-    allSymbols <- data.frame(Ticker = Symbols, stringsAsFactors = FALSE)
-    r <- merge(allSymbols, r, by = "Ticker", all.x = TRUE)
-  }
-
-  rownames(r) <- r$Ticker
-  std.cols <- c("Trade Time", "Open", "High", "Low", "Last", "Volume")
```

Comment:
Why did you decide to remove this code that you added in commits 49829d7231e6d5408c572a0462a1a63f1801c82f and 333fbc4e2882542597e4ce5cdb46a4b449b1514d?

Reply:
i moved all row name handling up to the master `getQuote` function. I thought it was cleaner to have the sub-functions return Symbol as a column and have the master function do the outer join and row names only in 1 place. If prefer the row names handling to stay in the sub functions, I can reverse that

Reply:
Thanks for explaining. I understood that you moved the column name and symbol handling to the main function. But the removed code did more than rename columns and add row names for the symbols.

So I'm asking why you removed that additional functionality. You wrote the initial version of the function, so I'm not sure why/if that additional functionality was needed or used. But it could have returned more columns than the new version does.

Reply:
weird, the github web interface was only showing lines 388-389 on this comment, but the email shows more. I assume you're asking about the 

```
colnames(r) <- gsub("(^[[:alpha:]])", "\\U\\1", colnames(r), perl = TRUE)
r[, "Trade Time"] <- r[, "LastSaleTimestamp"]
r[, "LastSaleTimestamp"] <- NULL

```
these lines were my lazy attempt at fixing the column names to upper case the first letter and to rename the lastsaletimestamp column. it was overly complex and i think the new code is more explicit about intent and will be easier to maintain